src/osd/OSD.h

```diff
@@ -1762,9 +1761,7 @@ class OSD : public Dispatcher,
 	string lock_name, string ordering_lock,
```

Comment:
There is no need for 2  lock names now.  I rebased and added a commit to address this.

See https://github.com/dzafman/ceph/tree/wip-sage-lock